README.md

```diff
@@ -11,6 +11,7 @@ A collection of android security related resources.
 ### Online Analyzers
 
 1. [AndroTotal](http://andrototal.org/)
+2. [AppSweep by Guardsquare](https://appsweep.guardsquare.com/) - Free, fast android application security testing for developers
```

Comment:
Thanks for the contribution. Please add it towards the end, above all the dead options.

Reply:
The new version should be fine